Okta.Xamarin/Okta.Xamarin/UserInfo.cs

```diff
@@ -0,0 +1,54 @@
+﻿// <copyright file="UserInfo.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Security.Claims;
+using Newtonsoft.Json;
+using Okta.Xamarin.Models;
+
+namespace Okta.Xamarin
+{
+    public class UserInfo : Serializable
+    {
+        [JsonProperty("sub")]
+        public string Sub { get; set; }
```

Comment:
I think you can create custom claims from the Okta dashboard and choose to get them from the userinfo endpoint. You might want to consider that scenario, otherwise, devs won't have access to them.

Reply:
I wondered about that; I'm glad you mentioned it.  I'll investigate more and test.

Reply:
OKTA-371439 added to ensure proper mapping of all claims to ClaimsPrincipal